bundle/edu.gemini.ags/src/main/java/edu/gemini/ags/gems/GemsGuideStarSearchOptions.java

```diff
@@ -30,7 +29,7 @@
         UCAC4_GEMINI("UCAC4", "UCAC4 at Gemini"),
         ;
 
-        public static CatalogChoice DEFAULT = UCAC4_GEMINI;
+        public static CatalogChoice DEFAULT = PPMXL_GEMINI;
```

Comment:
Should this be Gaia?

Reply:
Yes, I think I need to use Gaia now.  I found it odd that we define a `DEFAULT` but then explicitly mention `PPXML` everywhere.